TestAssets/TestProjects/PJLibWithMultipleFrameworks/project.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We already have https://github.com/dotnet/cli/tree/rel/1.0.0/TestAssets/TestProjects/TestLibraryWithMultipleFrameworks. Will that work for you?